citi-onboarding/CJR-onepage
# Fix/footer

## What I did

- Fixed  footer
- Fixed the axios/api

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   2.0. `cd server`
   2.1. `node index.js`

3. Open a new terminal:
   3.0. `cd client`
   3.1. `git checkout develop`
   3.2. `git pull`
   3.3. `git checkout fix/contacts`
   2.4. `git pull fix/contacts`
   2.5. `npm start`
   
  4. See if the footer is working in web and mobile
  
  OBS.: In the mobile the width seems to be smaller than the screen width, but it's a bug at cases and contact, if you comment the cases and contact section at App.js it will work fine.
